File: easy/58_length_of_last_word.py
# easy
# O(n)

class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        ss = s.split(' ')
        ans=''
        for word in ss:
            if word!='':
                ans = word
        return len(ans)

File: 58.length-of-last-word.go
/*
 * @lc app=leetcode id=58 lang=golang
 *
 * [58] Length of Last Word
 */

// @lc code=start
package main

import "strings"

func lengthOfLastWord(s string) int {
	ans := 0
	words := strings.Split(s, " ")
	for _, w := range words {
		if len(w) == 0 {
			continue
		}
		ans = len(w)
	}
	return ans
}

// @lc code=end
